pages/index.vue

```diff
@@ -171,7 +171,8 @@ export default class extends QiskitPage {
 
   &__paragraph {
     @include type-style('body-long-01');
-    max-width: 6 * $column-size-large;
+    color: $gray-60;
+    max-width: 6.25 * $column-size-large;
```

Comment:
Please, revert it to 6. This is the size for the paragraphs in the landing page (for large size) as agreed per offline conversations.